app/routes/search/agendaitems.js

```diff
@@ -69,6 +68,13 @@ export default class AgendaitemSearchRoute extends Route {
         entry.shortTitle = entry.shortTitle.replaceAll(highlightShortTitleEmsRemoved, agendaitem.highlight.shortTitle[i]);
       }
     } 
+    if ((agendaitem.highlight && agendaitem.highlight.title) && agendaitem.highlight.title[0]?.split(" ").length < agendaitem.attributes.title?.split(" ").length ) {
```

Comment:
Does this condition need to be more than just `agendaitem.highlight?.title`? Seems like a redundant optimization to me and reduces readability.